### PR TITLE
removed shadow

### PR DESCRIPTION
**Pull request by injas-bigbinary**
@Tsudhishnair _a Please check

### CODE REVIEW

File: src/components/Accordion.js [hunk 1]
```diff
@@ -88,7 +88,8 @@ export const Accordion = props => {
       width="100%"
       {...rest}
     >
-      <Card
+      <Touchable
```
Comment (Tsudhishnair) [inline src/components/Accordion.js:91]: @injas-bigbinary We need to use the Touchable component from the react-native. The one we have in the component doesn't support nested clicks.

**Review by Tsudhishnair — CHANGES_REQUESTED** [review]

File: src/components/Accordion.js (hunk 1 above)
Comment (injas-bigbinary) [reply src/components/Accordion.js]: @Tsudhishnair Updated

**Comment by injas-bigbinary** [comment]
@Tsudhishnair _a

**Comment by Tsudhishnair** [comment]
@injas-bigbinary _a LGTM. Please fix the conflict and merge.

**Pull request merged**